### PR TITLE
dbiscoIntuit adds hometown of San Diego

### PR DESCRIPTION
**Pull request by dbiscoIntuit**
Hello 👋 

I'm proposing adding San Diego to the list of hometowns, but I have some questions:

1. If you haven't been to San Diego before -- what are you looking forward to?
2. If you _have_ been to San Diego -- what did you enjoy the most? Or the least?
3. What kind of food does everyone like? I can make restaurant recommendations based on that.
4. Hey @KhaiMai2016 -- what kinds of things does your sons like to do?

### CODE REVIEW

**Comment by huyto** [comment]
i've been to San Diego couple times but only one day trip. I enjoyed the beach, ocean view

**Comment by KhaiMai2016** [comment]
1. trying different kind of Brewery
4. Watching movies, swimming, play with toys etc...

File: dbiscoIntuit-sanDiego.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# San Diego, CA
+
+### Restaurants
+
```
Comment (KhaiMai2016) [inline dbiscoIntuit-sanDiego.md]: Which is the best Thai Restaurant near our Office

**Comment by dbiscoIntuit** [comment]
> trying different kind of Brewery
> Watching movies, swimming, play with toys etc...

Sorry @KhaiMai2016 , I don't drink beer but there are alot of breweries in Miramar, 15 minutes South of Intuit.

File: dbiscoIntuit-sanDiego.md (hunk 1 above)
Comment (dbiscoIntuit) [inline dbiscoIntuit-sanDiego.md]: > Which is the best Thai Restaurant near our Office

I like going Sab-E-Lee because of their Mango sticky rice.

**Comment by dbiscoIntuit** [comment]
> i've been to San Diego couple times but only one day trip. I enjoyed the beach, ocean view

If you are a beer drinker, we have many micro-breweries.

**Comment by dbiscoIntuit** [comment]
@KhaiMai2016 and @huyto thanks for the feedback! I've added that information in commit 2b51cee

**Comment by githubteacher** [comment]
@dbiscoIntuit 👍  go ahead and 🚢

**Pull request merged**